File: ios/RNBridgeApp/CustomViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(CustomViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(customText, NSString)
RCT_EXTERN_METHOD(setCustomText:(nonnull NSNumber *)node text:(NSString *)text)
@end

File: ios/CustomViewManager.swift
import Foundation
import UIKit
import React

@objc(CustomViewManager)
class CustomViewManager: RCTViewManager {
    // Add this method
    override class func moduleName() -> String! {
        return "CustomView"
    }
    
    override func view() -> UIView! {
        return CustomView()
    }
    
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
    
    @objc
    func setCustomText(_ node: NSNumber, text: String) {
        DispatchQueue.main.async {
            if let view = self.bridge?.uiManager.view(forReactTag: node) as? CustomView {
                view.setCustomText(text)
            }
        }
    }
}
